routes/products.js

```diff
@@ -1,8 +1,26 @@
 const express = require('express');
+const wdk = require('wikidata-sdk');
+const rp = require('request-promise');
+
 const router = express.Router();
 const {ObjectID} = require('mongodb');
 const {Product} = require('./../models/product');
 
+// POST request to add a document to the database
+router.post('/', (req, res) => {
```

Comment:
Because this request only adds a documents to the database this should be a PUT operation rather than a POST op.